docs/developer-laptop-setup/install-vs-code.md

```diff
@@ -12,6 +12,21 @@ In line with Defra's digital standards for JavaScript, StandardJS coding standar
   `npm install standard`
 1. Set auto fix on save by adding `"standard.autoFixOnSave": true` to settings.json
 
+## Install C# Extension
+Add C# extension to enable .Net Core development.
+
+https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp
+
+## Install .Net Core SDK
+Follow Microsoft's setup guide according to your distro.  
+
+https://docs.microsoft.com/en-gb/dotnet/core/install/linux-package-manager-ubuntu-1804
```

Comment:
Might be an idea to put this first, so that error messages are avoided in Visual Studio Code

Reply:
Good idea.  Done.